helios-services/src/main/java/com/spotify/helios/rollingupdate/RollingUpdateOpFactory.java

```diff
@@ -78,10 +83,18 @@ public RollingUpdateOp nextTask(final List<ZooKeeperOperation> operations) {
       // Emit an event signalling that we're DONE!
       events.add(eventFactory.rollingUpdateDone(deploymentGroup));
     } else {
+      final DeploymentGroupTasks.Builder taskBuilder = tasks.toBuilder()
```

Comment:
in the above ^ if block a task failed event should be emitted if `failedTarget` is non-null, right?